proposals/299-ip-failure-count.txt

```diff
@@ -136,7 +136,7 @@ Ticket: https://trac.torproject.org/projects/tor/ticket/27491
    The initial value of the failure points in the scenarios described in this
    section would be:
 
-      * IPv4 Faulure Points: Count the number of IPv6-capable relays
+      * IPv4 Failure Points: Count the number of IPv6-capable relays
```

Comment:
I don't think "failure points" is a good name for these counters.

They are currently a combination of:
1. the number of IPv6 and IPv4 relays in the consensus
2. the number of successful connections over IPv6 and IPv4
3. the number of failed connections over IPv4 and IPv6, with different numbers of points for different types of failures.

But we also want to count only the number of IPv4 and IPv6 failures, so we can mark guards as down in Section 8.

Maybe we should call them "affinity points", or a similar name?
So:
* IPv4 failure points becomes IPv6 affinity points.
* IPv6 failure points becomes IPv4 affinity points.

Reply:
I think affinity points is a better name as well.

I initially used "failure points" because older versions of this proposal (and pre-Prop299 code) used this term, but since now we also count successes, "affinity points" is a better name.

Reply:
Thanks, let's fix that after Nick does his review.